docs/Documentation/Drivers/Classic/OutputFormatting.md

```diff
@@ -56,7 +56,17 @@ _type_ = (for `BINARY` output files) data type code. Must be one of:
 _multiplier_ = (for `BINARY` output files) factor to multiply the data by before writing, to increase precision compared to not using the multiplier.
   - `*` = use the default multiplier for this variable
 
-Here's an example. To specify 2 output files, named "wbal" and "ebal", and containing water balance and energy balance terms, respectively, you could do something like this:
+_aggtype_ = Aggregation method to use for temporal aggregation. Valid options for aggtype are:
+
+  - `AGG_TYPE_DEFAULT` = default aggregation type for variable
+  - `AGG_TYPE_AVG` = average over aggregation window
+  - `AGG_TYPE_BEG` = beginning of aggregation window
+  - `AGG_TYPE_END` = end of aggregation window
+  - `AGG_TYPE_MAX` = maximum in aggregation window
+  - `AGG_TYPE_MIN` = minimum in aggregation window
+  - `AGG_TYPE_SUM` = sum over aggregation window
+
+Here's an example. To specify 2 output files, named `wbal` and `ebal`, and containing water balance and energy balance terms, respectively, you could do something like this:
 
```

Comment:
this example seems incomplete, and I don't think it's super informative (since writing variables to particular output files hasn't changed). Maybe include a short example using one of the new `aggtype` options.